frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts

```diff
@@ -1,11 +1,12 @@
-import { K8sResourceKind } from '@console/internal/module/k8s';
+import { K8sResourceKind, Taint } from '@console/internal/module/k8s';
 
 export const minSelectedNode = 3;
-export const taintObj = {
+export const OCS_TAINT: Taint = {
```

Comment:
This isn't following the convention in the same file (`ocsRequestData` and `infraProvisionerMap`). I have a preference for lower case here.

```suggestion
export const ocsTaint: Taint = {
```